apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/CreateOrder/Actions/OrderActions/OrderActions.tsx

```diff
@@ -92,16 +94,26 @@ const RefreshIcon = ({onPress, disabled}: {onPress: () => void; disabled: boolea
       onPressOut={() => setIsActive(false)}
     >
       <Animated.View style={getRotationStyle()}>
-        <Icon.Refresh size={28} color={disabled ? COLORS.DISABLED : ''} active={isActive} />
+        <Icon.Refresh size={28} color={disabled ? colors.disabled : ''} active={isActive} />
```

Comment:
not comfortable with this `: ''` for color, cuz `'' ?? 1` is `''`, can you apply the enabled color?